<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Abonnementwindow</class>
 <widget class="QMainWindow" name="Abonnementwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Espace Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#etudiantwindow {
background:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);

}

QLabel {
color :  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
font-weight: bold;
font-size: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>info Abonnement </string>
     </attribute>
     <widget class="QTableView" name="tableAbonnement">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>171</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>insérez id Abonnement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>89</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ush">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>acheter en ligne sur Tayara.tn</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>info Cadeau</string>
     </attribute>
     <widget class="QTableView" name="tableView_Cadeau2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_notezCadeau">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>notez Cadeau</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_note">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>310</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>insérez le jour</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_inserezCadeau"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_afficherCadeau">
         <property name="text">
          <string>afficher</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tableAbonnement</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tableView_Cadeau2</tabstop>
  <tabstop>pushButton_notezCadeau</tabstop>
  <tabstop>spinBox_note</tabstop>
  <tabstop>lineEdit_inserezCadeau</tabstop>
  <tabstop>pushButton_afficherCadeau</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
